app/main.py

```diff
@@ -1,3 +1,63 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import math
+from app.purchase import Purchase
+
+
+with open("app/config.json", "r") as data_customers:
+    data = json.load(data_customers)
+
+
+class GetMostCheapShop(Purchase):
+    def __init__(self, data_file: dict) -> None:
+        super().__init__(data_file)
+
+    def fuel_costs(self, shop: dict, customer: dict) -> None:
+        fuel_price = self.data["FUEL_PRICE"]
+        car_consumption = customer["car"]["fuel_consumption"]
+        distance = math.dist(customer["location"], shop["location"])
+        return round(distance * 2 / 100 * car_consumption * fuel_price, 2)
+
+    def how_much_costs_trip(self, customers: dict) -> dict:
+        customer_chart_milk = customers["product_cart"]["milk"]
```

Comment:
better say ```cost_of_the_trip``` or ```trip_cost``` 
Or even create a ```class Trip``` and then have a method or even a property as ```cost```